src/cpp/rtps/participant/RTPSParticipantImpl.h

```diff
@@ -49,6 +49,7 @@
 #include <fastrtps/utils/shared_mutex.hpp>
 
 #include "../flowcontrol/FlowControllerFactory.hpp"
+#include <fastdds/builtin/type_lookup_service/TypeLookupManager.hpp>
```

Comment:
It is not necessary as it has been forward declared. It can be removed.

Reply:
Removed in [ee520e4](https://github.com/eProsima/Fast-DDS/pull/4177/commits/ee520e48f15c205983ef93344d678cca12d35d0b)